Formula/symengine.rb

```diff
@@ -0,0 +1,30 @@
+class Symengine < Formula
+  desc "Fast symbolic manipulation library written in C++"
+  homepage "https://sympy.org"
+  url "https://github.com/symengine/symengine/releases/download/v0.8.1/symengine-0.8.1.tar.gz"
+  sha256 "41eb6ae6901c09e53d7f61f0758f9201e81fc534bfeecd4b2bd4b4e6f6768693"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "gmp"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <symengine/expression.h>
+      using SymEngine::Expression;
+      int main() {
+        Expression x("x");
+        auto ex = pow(x+sqrt(Expression(2)), 6);
```

Comment:
Can we rewrite this to produce output we can check? Or at least, make this `return 1` if the contents of `ex` aren't what's expected.

Reply:
done